components/radio/stories/radio.stories.js

```diff
@@ -40,15 +40,20 @@ export default {
 		isChecked,
 		isDisabled,
 		isReadOnly,
+		isHovered,
+		isFocused,
 	},
 	args: {
 		rootClass: "spectrum-Radio",
 		size: "m",
 		label: "Label",
 		isEmphasized: false,
 		isChecked: false,
+		isInvalid: false,
```

Comment:
We probably don't need this `isInvalid` arg anymore. I think it's causing an uncategorized argument in the controls:

<img width="1011" alt="Screenshot 2025-03-17 at 2 13 59 PM" src="https://github.com/user-attachments/assets/4c7cc150-e6d7-4ca6-8608-a1de3a3558fd" />